frontend/server/src/DAO/TeamsGroupRoles.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace OmegaUp\DAO;
+
+/**
+ * TeamsGroupRoles Data Access Object (DAO).
+ *
+ * Esta clase contiene toda la manipulacion de bases de datos que se necesita
+ * para almacenar de forma permanente y recuperar instancias de objetos
+ * {@link \OmegaUp\DAO\VO\TeamsGroupRoles}.
+ *
+ * @access public
+ * @package docs
+ */
+class TeamsGroupRoles extends \OmegaUp\DAO\Base\TeamsGroupRoles {
+    public static function deleteAllTeamGroupsForContest(int $aclId): int {
```

Comment:
```suggestion
    public static function deleteAllTeamGroupsForAclId(int $aclId): int {
```

a menos que se le pueda pasar un objeto de concurso de verdad para que concuerde lo que dice y lo que hace.

Reply:
Listo